components/builder-api-client/src/lib.rs

```diff
@@ -412,37 +400,31 @@ impl Client {
     /// # Failures
     ///
     /// * Remote API Server is not available
-    pub fn job_group_promote_or_demote<T: AsRef<str> + serde::Serialize>(
-        &self,
-        group_id: u64,
-        idents: &[T],
-        channel: &ChannelIdent,
-        token: &str,
-        promote: bool,
-    ) -> Result<()> {
+    pub fn job_group_promote_or_demote<T: AsRef<str> + serde::Serialize>(&self,
+                                                                         group_id: u64,
+                                                                         idents: &[T],
+                                                                         channel: &ChannelIdent,
+                                                                         token: &str,
+                                                                         promote: bool)
+                                                                         -> Result<()> {
         let json_idents = json!(idents);
```

Comment:
This feels a little extreme in terms of width.

Reply:
Agreed; can we tune this down a little bit to get the nice stuff but not go this far?